### PR TITLE
Fix build issues and resource names clash

### PR DESCRIPTION
**Pull request by Vandalko**
Fixes:
 - https://github.com/googlemaps/android-maps-utils/issues/259
 - https://github.com/googlemaps/android-maps-utils/issues/252

### CODE REVIEW

**Comment by Vandalko** [comment]
I signed it!

File: library/build.gradle [hunk 1]
```diff
@@ -48,7 +50,7 @@ android {
 task instrumentTest(dependsOn: connectedCheck)
 
 task apklib(type: Zip) {
```
Comment (Vandalko) [inline library/build.gradle]: does anyone use apklib nowadays ? Eclipse users ?

**Comment by Vandalko** [comment]
build failed because of mis-configured Travis CI
`failed to find Build Tools revision 23.0.2`

File: library/build.gradle [hunk 2]
```diff
@@ -23,7 +23,9 @@ dependencies {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.0"
+    buildToolsVersion "23.0.2"
```
Comment (barbeau) [inline library/build.gradle]: You also need to bump `ANDROID_BUILD_TOOLS_VERSION` to match in `.travis.yml` - https://github.com/googlemaps/android-maps-utils/blob/master/.travis.yml#L15.

Comment (Vandalko) [inline library/build.gradle]: @barbeau thx for the hint - all checks have passed now

File: build.gradle [hunk 3]
```diff
@@ -4,9 +4,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0-beta5'
+        classpath 'com.android.tools.build:gradle:2.1.0-alpha4'
```
Comment (markmcd) [inline build.gradle]: Are you using something from the latest alpha release? If not, I'd prefer to leave this dependent on the theoretically more stable beta* releases if we can't use the full releases.

File: library/src/com/google/maps/android/ui/RotationLayout.java [hunk 4]
```diff
@@ -26,7 +26,7 @@
  * <p/>
  * May not work with padding.
  */
-class RotationLayout extends FrameLayout {
+public class RotationLayout extends FrameLayout {
```
Comment (markmcd) [inline library/src/com/google/maps/android/ui/RotationLayout.java]: This looks unrelated to the resource prefix changes.

**Comment by markmcd** [comment]
Thanks for taking a look at this. In future, feel free to send through multiple small pull requests, even if one is dependent on another. It can make reviews much easier for us.

File: library/src/com/google/maps/android/ui/RotationLayout.java (hunk 4 above)
Comment (Vandalko) [inline library/src/com/google/maps/android/ui/RotationLayout.java]: it was a lint error - since this class is used in `layout.xml` file, lint requires it to be public

File: build.gradle (hunk 3 above)
Comment (Vandalko) [inline build.gradle]: @markmcd I'm not useing 2.1.0 features directly, but latest Android Studio 2.1 preview requires `2.1.0-alpha4` gradle plugin.

I can revert to `2.0.0-beta7` because it's not critical. Pls let me know if I should.

File: library/src/com/google/maps/android/ui/RotationLayout.java (hunk 4 above)
Comment (Vandalko) [inline library/src/com/google/maps/android/ui/RotationLayout.java]: check `text_bubble.xml`

Comment (markmcd) [inline library/src/com/google/maps/android/ui/RotationLayout.java]: OK, SGTM

File: build.gradle (hunk 3 above)
Comment (markmcd) [inline build.gradle]: Yes, please do. If you're using an unstable version of Android Studio, it will suggest an unstable version of gradle. I'd prefer to keep this as stable as possible.

**Comment by Vandalko** [comment]
Downgraded Android Gradle plugin to latest beta

**Comment by markmcd** [comment]
Merged in 29490ad. Thanks again!
